chart-values-json-schema-validation.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+import requests
+import urllib3
+from jsonschema import validate
+from ruamel.yaml import YAML
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+SCHEMA_BASE_URL = "https://raw.githubusercontent.com/vl-kp/json-schema/main"
+GITOPS_DIR = "gitops"
+
+
+def download_schema_json(version):
+    """
+    Download json schema for chart version.
+    """
+    version_url = f"{SCHEMA_BASE_URL}/version.json"
+    version_json = requests.get(version_url, timeout=10, verify=False).json()
+    return_version = next(
+        (
+            item["JSONSchemaVersion"]
+            for item in version_json["ranges"]
+            if item["end"] >= version >= item["start"]
+        ),
+        None,
+    )
+    if return_version:
+        schema_url = f"{SCHEMA_BASE_URL}/v{return_version}/schema-platform-managed-chart-strict.json"
+        schema_json = requests.get(schema_url, timeout=10, verify=False).json()
+        return schema_json
+    return None
+
+
+def delete_error_files(directory="."):
+    """
+    Delete error files.
+    """
+    for filename in os.listdir(directory):
+        if filename.startswith("error-merged-values-") and filename.endswith(".yaml"):
```

Comment:
You can use a glob to directly select this files

Reply:
replace with glob